src/util.js

```diff
@@ -0,0 +1,34 @@
+/**
+   Public util functions for the SDK.
+ */
+
+const isObject = val => Object.prototype.toString.call(val) === '[object Object]';
+
+/**
+   Serialize a single attribute in an object query parameter.
+*/
+const serializeAttribute = attribute => {
+  if (isObject(attribute)) {
+    throw new Error('Nested object in query parameter.');
+  } else if (Array.isArray(attribute)) {
+    return attribute.join(',');
+  } else {
+    return attribute;
+  }
+};
+
+/**
+   Serializes an object into a Flex API query parameter value format:
+   key1:val1;key2:val2;key3:array,values,comma,separated.
+*/
+// eslint-disable-next-line import/prefer-default-export
+export const objQueryParam = obj => {
+  if (!isObject(obj)) {
+    throw new Error('Parameter not an object.');
+  }
+
+  return Object.entries(obj)
+    .filter(([, v]) => !!v)
+    .map(([k, v]) => `${k}:${serializeAttribute(v)}`)
+    .join(';');
```

Comment:
Comment/Change request maybe: Params serializer calls `encode(v)` to all serialized values. I guess here we should do the same? https://github.com/sharetribe/flex-sdk-js/blob/master/src/params_serializer.js#L49

Reply:
@rap1ds I looked into the `encode(v)` function and how it's used. The reason why I left it out was that the string value produced by the object util function goes through the pram serializer and the encoding is performed there.